Sources/SwiftDocC/Infrastructure/Workspace/FeatureFlags+Info.swift

```diff
@@ -0,0 +1,86 @@
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension DocumentationBundle.Info {
+    /// A collection of feature flags that can be enabled from a bundle's Info.plist.
+    ///
+    /// This is a subset of flags from ``FeatureFlags`` that can influence how a documentation
+    /// bundle is written, and so can be considered a property of the documentation itself, rather
+    /// than as an experimental behavior that can be enabled for one-off builds.
+    ///
+    /// ```xml
+    /// <key>CDExperimentalFeatureFlags</key>
+    /// <dict>
+    ///     <key>ExperimentalOverloadedSymbolPresentation</key>
+    ///     <true/>
+    /// </dict>
+    /// ```
+    public struct BundleFeatureFlags: Codable, Equatable {
+        /// Whether or not experimental support for combining overloaded symbol pages is enabled.
+        ///
+        /// This feature flag corresponds to ``FeatureFlags/isExperimentalOverloadedSymbolPresentationEnabled``.
+        public var experimentalOverloadedSymbolPresentation: Bool?
+
+        public init(experimentalOverloadedSymbolPresentation: Bool? = nil) {
+            self.experimentalOverloadedSymbolPresentation = experimentalOverloadedSymbolPresentation
+            self.unknownFeatureFlags = []
+        }
+
+        /// A list of decoded feature flag keys that didn't match a known feature flag.
+        public let unknownFeatureFlags: [String]
+
+        enum CodingKeys: String, CodingKey {
+            case experimentalOverloadedSymbolPresentation = "ExperimentalOverloadedSymbolPresentation"
+        }
+
+        struct AnyCodingKeys: CodingKey {
+            var stringValue: String
+
+            init?(stringValue: String) {
+                self.stringValue = stringValue
+            }
+
+            var intValue: Int? { nil }
+            init?(intValue: Int) {
+                return nil
+            }
+        }
+
+        public init(from decoder: any Decoder) throws {
+            let values = try decoder.container(keyedBy: AnyCodingKeys.self)
+            var unknownFeatureFlags: [String] = []
+
+            for flagName in values.allKeys {
+                if let codingKey = CodingKeys(stringValue: flagName.stringValue) {
+                    switch codingKey {
+                    case .experimentalOverloadedSymbolPresentation:
+                        self.experimentalOverloadedSymbolPresentation = try values.decode(Bool.self, forKey: flagName)
+                    }
+                } else {
+                    unknownFeatureFlags.append(flagName.stringValue)
+                }
+            }
+
+            self.unknownFeatureFlags = unknownFeatureFlags
+
+            if let overloadsFlag = self.experimentalOverloadedSymbolPresentation {
+                FeatureFlags.current.isExperimentalOverloadedSymbolPresentationEnabled = overloadsFlag
```

Comment:
Why does this need to update `FeatureFlags.current` here in addition to the call to `FeatureFlags.current.loadFlagsFromBundle(bundleFlags)` in `register`?

Reply:
Because i forgot to remove this code 😅 I can take it out.